src/containers/AdminCampaignStats/TopLineStats.jsx

```diff
@@ -1,49 +1,41 @@
 import { gql } from "@apollo/client";
-import { css, StyleSheet } from "aphrodite";
+import { Grid } from "@material-ui/core";
 import PropTypes from "prop-types";
 import React from "react";
 
-import theme from "../../styles/theme";
 import { withQueries } from "../hoc/with-operations";
 import CampaignStat from "./CampaignStat";
 
-const styles = StyleSheet.create({
-  container: {
-    ...theme.layouts.multiColumn.container,
-    marginBottom: 40,
-    justifyContent: "space-around",
-    flexWrap: "wrap"
-  },
-  flexColumn: {
-    flex: 1,
-    textAlign: "right",
-    display: "flex"
-  },
-  spacer: {
-    marginRight: 20
-  }
-});
-
 export const TopLineStats = (props) => {
```

Comment:
Suggestion: Add `TopLineStatsProps` and add it as a type for the function

Reply:
Going to merge without this because it's probably best to do this one as part of a separate PR that also moves the queries into codegen and eliminates the `withQueries` props. Intended this PR to be mostly focused on the UI changes